src/Kiota.Builder/CodeDOM/IAccessibleElement.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Kiota.Builder.CodeDOM;
+public interface IAccessibleElement
```

Comment:
nit: document this interface? When should it be implemented? What behavior is it abstracting?

Reply:
Will update the documentation. 
Interface is already implemented by allowing the agnostic access of the `AccessModifier` property already present in DOM elements such as `CodeProperty` and `CodeMethod`